staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go

```diff
@@ -93,6 +94,9 @@ func newWithBackoff(subjectAccessReview authorizationclient.SubjectAccessReviewI
 		authorizedTTL:       authorizedTTL,
 		unauthorizedTTL:     unauthorizedTTL,
 		initialBackoff:      initialBackoff,
+		// For now, we need to leave this as true for backwards
+		// compatibility. Consider changing the default to false.
+		failOpen: true,
```

Comment:
`decisionOnError: authorizer.DecisionNoOpinion`, and I don't think we'd change that as the default